recipes/graphql/app/resolvers.js

```diff
@@ -0,0 +1,7 @@
+const resolvers = {
+  Query: {
+    hello: (_, { value }) => value !== 42 ? 'Hello world!' : (() => { throw new Error('42') })(),
```

Comment:
No need to complicate things...
```suggestion
    hello: (_, { value }) => 'Hello world!',
```